cvat/apps/annotation/builtin/pascal_voc/dumper.py

```diff
@@ -0,0 +1,33 @@
+from pascal_voc_writer import Writer
```

Comment:
I now see that the library is very limited and simple. Probably you want to remove it.